namespaces/cloud-platform-test-1.k8s.integration.dsd.io/traefik/traefik-ui.yaml

```diff
@@ -15,10 +15,9 @@ apiVersion: extensions/v1beta1
 kind: Ingress
```

Comment:
Have you tested these changes? This `Ingress` looks invalid. As per my original comment we shouldn't expose it so please remove the `Ingress` object.